[2025-02-02T07:56:47.287+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T07:56:47.296+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: idp.load_requests manual__2025-02-02T07:56:45.054942+00:00 [queued]>
[2025-02-02T07:56:47.302+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: idp.load_requests manual__2025-02-02T07:56:45.054942+00:00 [queued]>
[2025-02-02T07:56:47.302+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-02T07:56:47.310+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_requests> on 2025-02-02 07:56:45.054942+00:00
[2025-02-02T07:56:47.315+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=484) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T07:56:47.316+0000] {standard_task_runner.py:72} INFO - Started process 485 to run task
[2025-02-02T07:56:47.317+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'idp', 'load_requests', 'manual__2025-02-02T07:56:45.054942+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/idp.py', '--cfg-path', '/tmp/tmpzcajf7jz']
[2025-02-02T07:56:47.317+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask load_requests
[2025-02-02T07:56:47.367+0000] {task_command.py:467} INFO - Running <TaskInstance: idp.load_requests manual__2025-02-02T07:56:45.054942+00:00 [running]> on host 3aa1f743f988
[2025-02-02T07:56:47.410+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='idp' AIRFLOW_CTX_TASK_ID='load_requests' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T07:56:45.054942+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T07:56:45.054942+00:00'
[2025-02-02T07:56:47.412+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-02T07:56:47.412+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-02T07:56:47.412+0000] {logging_mixin.py:190} INFO - Current task name:load_requests state:running start_date:2025-02-02 07:56:47.297232+00:00
[2025-02-02T07:56:47.412+0000] {logging_mixin.py:190} INFO - Dag name:idp and current dag run status:running
[2025-02-02T07:56:47.413+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T07:56:47.413+0000] {logging_mixin.py:190} INFO - MongoDB connection failed: MONGODB_URI is not set!
[2025-02-02T07:56:47.413+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-02-02T07:56:47.424+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T07:56:47.424+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=idp, task_id=load_requests, run_id=manual__2025-02-02T07:56:45.054942+00:00, execution_date=20250202T075645, start_date=20250202T075647, end_date=20250202T075647
[2025-02-02T07:56:47.428+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-02T07:56:47.428+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-02T07:56:47.429+0000] {logging_mixin.py:190} INFO - Dag name:idp queued_at:2025-02-02 07:56:45.065667+00:00
[2025-02-02T07:56:47.429+0000] {logging_mixin.py:190} INFO - Task hostname:3aa1f743f988 operator:PythonOperator
[2025-02-02T07:56:47.457+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T07:56:47.465+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T07:56:47.466+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
